setup.cfg

```diff
@@ -48,11 +48,15 @@ markers=
 filterwarnings=
     error
     ignore::pytest.PytestUnraisableExceptionWarning
+    ignore:Save attribute '_xsection' was not loaded.:RuntimeWarning
+    ignore::RuntimeWarning:desc.compute
```

Comment:
i dont think we need to ignore this anymore? We used to divide by zero and then overwrite the bad value but i think that was fixed

Reply:
Deleted it